### PR TITLE
#163304765 create test for api endpoints 

### PR DESCRIPTION
**Pull request by JosephNjuguna**
### What does this PR do??
this pr adds test that are failing for the API endpoints
### Description of the task to be completed??
add test that are supposed to fail  for the following views 

- [ ] user sign up

- [ ] user log in

- [ ] post meetup

- [ ] post question

- [ ] user rsvp
### how should this be manualy tested??
```
clone this repo
open the repo folder in your terminal and run the following command: `python -m pytest`
to run all the test
```
### Relevant pivotal tracker stories ?
https://www.pivotaltracker.com/n/projects/2235137 **#163304765**

### CODE REVIEW

**Comment by gik-hub** [comment]
since you are beginning with tests, they should first fail, after which you write their bear minimum code

**Comment by gik-hub** [comment]
I have also  realized that your tests are empty, you can try to have some sample data which you test with the app client in the test functions, Additionally, I think you need to re-look your speed as we are racing against time.

**Comment by JosephNjuguna** [comment]
let me add sample data to my test

**Comment by JosephNjuguna** [comment]
@TgeeT i have worked on your feedback . kindly review

File: app/test/v2/test_meetups.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+"""test for api endpoints"""
+# inbuilt modelues
```
Comment (oagutu) [inline app/test/v2/test_meetups.py:2]: Minor typo: `modules`

File: app/test/v2/test_meetups.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import json
+import unittest
+# local imports
+from app import create_app
+class MeetupTest(unittest.TestCase):
+    def setUp(self):
+        self.appliaction = create_app
```
Comment (oagutu) [inline app/test/v2/test_meetups.py:9]: Minor typo: `application`.
Also you are assigning a function reference and not actually the result of the function call

**Review by oagutu — COMMENTED** [review]
Good job. You may need to account for tokens for your tests when testing protected endpoints. This will however depend on when you'll implement authentication, and so may not be necessary at this point.

File: app/test/v2/test_meetups.py (hunk 1 above)
Comment (JosephNjuguna) [reply app/test/v2/test_meetups.py:2]: NOTED THAT. i will change on this in my next PR.

File: app/test/v2/test_meetups.py (hunk 2 above)
Comment (JosephNjuguna) [reply app/test/v2/test_meetups.py:9]: I change typo on that in my next pr

Comment (JosephNjuguna) [reply app/test/v2/test_meetups.py:9]: thanks for the feedback.
> Also you are assigning a function reference and not actually the result of the function call
 
i will work on this too
